package.json

```diff
@@ -222,7 +222,7 @@
     "eslint": "^8.56.0",
     "eslint-config-google": "^0.14.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-jsdoc": "^48.0.1",
+    "eslint-plugin-jsdoc": "^48.4.0",
```

Comment:
This seems unrelated to the typo fix - any reason for updating this?

Reply:
This was added to the PR in the commit I had run the format command.

I can remove this from the PR.